nixos/modules/programs/partition-manager.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
```

Comment:
What value does this service add over just adding the package to your closure?

Reply:
That's from the PR https://github.com/NixOS/nixpkgs/pull/114016, which is required to make calamares build and is therefore included in this PR. The issue is that part of kpmcore is now a daemon called through DBus, so it needs a module to be register it on the bus. (grrr)

Reply:
Right, I see the `services.dbus.packages`, and I see it has become a pattern to create modules for this as well as `systemd.packages`.

We don't have it yet, but we should tag packages as providing dbus and/or systemd files instead.